libraries/package-extractor/src/PackageExtractor.ts

```diff
@@ -646,6 +693,32 @@ export class PackageExtractor {
       // and there is no exclude filter, then we know that the file is not excluded. If it is included
       // and there is an exclude filter, then we need to check for a match.
       return !isIncluded || !!excludeFilters?.some((m) => m.match(filePath));
+    };
+    // Function to filter files inside local project
+    const isFileExcludedForProject = (filePath: string): boolean =>
+      isFileExcluded(filePath, {
+        ...sourceProjectConfiguration
+      });
```

Comment:
```suggestion
      isFileExcluded(filePath, sourceProjectConfiguration);
```

The second parameter doesn't appear to get modified, so it doesn't need to be cloned.